examples/langserve-sample/packages/rag_vector_store/rag_vector_store/chain.py

```diff
@@ -0,0 +1,95 @@
+import os
+from typing import List
+
+from langchain.schema import format_document
+from langchain_core.documents import Document
+from langchain_core.output_parsers import StrOutputParser
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.prompts.prompt import PromptTemplate
+from langchain_core.pydantic_v1 import BaseModel
+from langchain_core.runnables import (
+    ConfigurableField,
+    RunnableParallel,
+)
+from langchain_core.runnables.utils import ConfigurableFieldSingleOption
+from langchain_openai import ChatOpenAI
+
+from zep_python import ZepClient
+from zep_python.langchain import ZepVectorStore
+
+ZEP_API_KEY = os.environ.get("ZEP_API_KEY", None)  # Required for Zep Cloud
```

Comment:
I don't think `None` is necessary,  `get` by default returns `None` if the key is not found.